apps/docs/stories/utils/documentation-components.tsx

```diff
@@ -49,3 +49,89 @@ export const SummaryElement = (props: { title: string, children: any }) => {
     </details>
   );
 };
+
+// If there's only one role, return <code>{role}</code>, 
+// otherwise return <code>{role1} or {role2}</code>
+const getComponentRole = ({ actions, entity, identification }: AuthorizationProps) => {
+  const getCodeForRole = (role: string) => {
+    return <code>{`${role}:${entity}:${identification}`}</code>
+  };
+
+  if (actions.length === 1) {
+    return getCodeForRole(actions[0]);
+  }
+
+  return (
+    <>{getCodeForRole(actions[0])} or {getCodeForRole(actions[1])}</>
+  );
+};
+
+const AuthTokenDescription = () => (
+  <>
+    <strong>Auth Token</strong >: These tokens are generated by your backend services using the {' '}
+    <a href="https://docs.justifi.tech/apiFundamentals/authentication">
```

Comment:
Could you please instead link to `https://docs.justifi.tech/api-spec#tag/API-Credentials/operation/CreateAccessToken` and call it `Create Access Token API`?